examples/plot_fpca.py

```diff
@@ -25,7 +25,22 @@
 # to explore datasets and obtain conclusions about said dataset using this
 # technique.
 #
-# First we are going to fetch the Berkeley Growth Study data. This dataset
+# First we will introduce what FPCA is and why it is commonly used in many
+# studies. Generally, functional data is represented as functions X(t) where
+# t might be time or some other continuous variable. In theory, these 
+# functions can have an infinite number of data points (they can be treated as
+# infinite dimension vector spaces) so the goal of FPCA is to reduce the 
+# complexity of studying those curves by finding a finite number of principal
+# components, which are the directions that capture the main modes of
+# variation across the function (the most important directions in which the
+# curves vary). FPCA can be though of as a basis expansion, but what 
+# distinguishes FPCA is that among all basis expansions that use K components
+# for a fixed K, the FPC expansion explains most of the variation in X.
+#
+# For more information abour FPCA and its objectives, see 
+# :footcite:ts:`wang+chiou+muller_2016_fpca`.
+# 
+# In second place, we are going to fetch the Berkeley Growth Study data. This dataset
```

Comment:
This is really the first step of the tutorial, the previous paragraph being a short explanation of the method. I would drop that commit.